MapboxNavigation/UIImage.swift

```diff
@@ -33,4 +33,29 @@ extension UIImage {
             (text as NSString).draw(in: rect.offsetBy(dx: 0, dy: (constrainedSize.height - font.lineHeight) / 2).integral, withAttributes: textFontAttributes)
         }
     }
+
+    func tint(_ tintColor: UIColor) -> UIImage {
+        let imageSize = size
+        let imageScale = scale
+        let contextBounds = CGRect(origin: .zero, size: imageSize)
+
+        UIGraphicsBeginImageContextWithOptions(imageSize, false, imageScale)
```

Comment:
By the way, `UIGraphicsImageRenderer` is also available on iOS 10 and above. It does have some shortcomings compared to the C-style `UIGraphicsBeginImageContextWithOptions(_:_:_:)` around transformations, but we’re already using it in several parts of the SDK.